client/src/test/scala/dwolla/cloudflare/AccessControlRuleClientSpec.scala

```diff
@@ -0,0 +1,345 @@
+package dwolla.cloudflare
+
+import cats.effect.Sync
+import com.dwolla.cloudflare._
+import com.dwolla.cloudflare.domain.model.Exceptions.UnexpectedCloudflareErrorException
+import com.dwolla.cloudflare.domain.model.{AccountId, AccountIdTag}
+import com.dwolla.cloudflare.domain.model.accesscontrolrules._
+import io.circe.Json
+import org.http4s.Status
+import org.http4s.client.Client
+import org.specs2.mutable.Specification
+import org.specs2.specification.Scope
+import io.circe.literal._
+import io.circe.parser.parse
+
+class AccessControlRuleClientSpec extends Specification {
+  trait Setup extends Scope {
+    val accountId: AccountId = shapeless.tag[AccountIdTag][String]("account-id")
+
+    val authorization = CloudflareAuthorization("email", "key")
+    val fakeService = new FakeCloudflareService(authorization)
+  }
+
+  private def parseJson(responseBody: String): Json = {
```

Comment:
Instead of using this `parseJson` method, it would be better to replace its use with Circe's JSON string interpolator. e.g.

```scala
scala> :paste
// Entering paste mode (ctrl-D to finish)

import io.circe.literal._

json"""{"my": "json", "is": "great"}"""

// Exiting paste mode, now interpreting.

import io.circe.literal._
res0: io.circe.Json =
{
  "my" : "json",
  "is" : "great"
}
```